indico/modules/rb/templates/emails/reservations/confirmation_email_to_user.txt

```diff
@@ -4,6 +4,11 @@
 {% block state -%}
 Your booking has been ACCEPTED.
 This is the final confirmation.
+
+{% if reservation.acceptance_reason %}
```

Comment:
I think you need `{%- ...}` here to swallow the whitespace in case there's no reason, and add the empty line inside the if's body instead. Probably also want `{%- endif -%}` below to avoid having an extra linebreak a the end.

Best to test with and without a commend and make sure there's no weird change in empty lines.